assets/src/onboarding-wizard/pages/template-mode/index.js

```diff
@@ -39,15 +39,19 @@ export function TemplateMode() {
 				<h1>
 					{ __( 'Template Modes', 'amp' ) }
 				</h1>
-				{ /* dangerouslySetInnerHTML reason: Injection of links. */ }
-				<p dangerouslySetInnerHTML={ {
-					__html: sprintf(
-						/* translators: placeholders are links to amp-wp.org website. */
-						__( 'Based on site scan results the AMP plugin provides the following choices. Learn more about the <a href="%1$s" target="_blank" rel="noreferrer noopener">AMP experience with different modes</a> and availability of <a href="%2$s" target="_blank" rel="noreferrer noopener">AMP components in the ecosystem</a>.', 'amp' ),
-						'https://amp-wp.org/documentation/getting-started/template-modes/',
-						'https://amp-wp.org/ecosystem/',
-					),
-				} } />
+				<p>
+					{
+						createInterpolateElement(
+							__( 'Based on site scan results the AMP plugin provides the following choices. Learn more about the <GettingStartedUrl>AMP experience with different modes</GettingStartedUrl> and availability of <EcosystemUrl>AMP components in the ecosystem</EcosystemUrl>.', 'amp' ),
```

Comment:
Nit: Instead of `Url` as the suffix, it would be better to have `Link` as the suffix.